### ISSUE TITLE
Crash on iOS 7.1

### ISSUE THREAD

**Issue by krazie99** [inamiy/YIFullScreenScroll#27]
If not use custom image on navigation-bar, crash on iOS 7.1 with this code.

YIFullScrenScroll.m

```
UIImageView* originalBackground = [bar.subviews objectAtIndex:0];
```

I don't know this is good solution. But, I changed it to 

```
UIImageView* originalBackground = nil;
if([bar.subviews count] > 0)
        originalBackground = [bar.subviews objectAtIndex:0];
```

Is this Correct?

**Comment by inamiy** [comment]
Thanks. I already fixed this issue in 5ea1cf331b7a4a64e8d68aa4924651aa29aa94b8 , so please use the latest commit.

**Comment by krazie99** [comment]
Thank you ;)
